tests/main/interfaces-removable-media/task.yaml

```diff
@@ -0,0 +1,50 @@
+summary: Ensure that the removable-media interface works.
+
+details: |
+    The removable-media interface allows to access to removable storage filesystems.
+environment:
+    TESTDIR: '/media/testdir'
```

Comment:
Can you please test the `/run/media/foo` variant as well? You can drop the `TESTFILE` variable and just use `TESTDIR/file` for simplicity.

Reply:
`/media` and `/run/media` split is a bit confusing as I stumbled upon this in #4285. There's a `--enable-merged-usr` snap-confine configure option, that results in either `/media` or `/run/media` being mounted into the child's namespace. IIRC there was some incosistency between Fedora/Arch/OpenSUSE on this matter. I think Arch ended up with `/media`, what made me doubt that the actual location is related to usr-split.